AppCreationScripts/sample.json

```diff
@@ -1,56 +0,0 @@
-{
-  "Sample": {
-    "Title": "Universal Windows Platform application signing in users with Microsoft Identity Platform and calling Microsoft Graph",
-    "Level": 200,
-    "Client": "UWP",
-    "Service": "Microsoft Graph",
-    "RepositoryUrl": "active-directory-dotnet-native-uwp-v2",
-    "Endpoint": "AAD v2.0"
-  },
-
-  /*
-    This section describes the Azure AD Applications to configure, and their dependencies
-  */
-  "AADApps": [
-    {
-      "Id": "uwpApp",
-      "Name": "UWP-App-calling-MSGraph",
-      "Kind": "UWP",
-      "RedirectUri": "https://login.microsoftonline.com/common/oauth2/nativeclient",
-      "RequiredResourcesAccess": [
-        {
-          "Resource": "Microsoft Graph",
-          "DelegatedPermissions": [ "User.Read" ]
-        }
-      ],
-      "ManualSteps": [
-        {
-          "Comment": "Navigate to the Manifest page and change 'signInAudience' to 'AzureADandPersonalMicrosoftAccount'."
-        },
-        {
-          "Comment": "Navigate to the Manifest page and change 'accessTokenAcceptedVersion' to 2."
```

Comment:
This is the right solution. Add a manual step, for instance described with an aka.ms link